### PR TITLE
rustup: add doc example

### PR DESCRIPTION
**Pull request by SebastianJL**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Hi, thanks for your contribution.

File: pages/common/rustup.md [hunk 1]
```diff
@@ -27,3 +27,7 @@
 - Run cargo build with a certain toolchain:
 
 `rustup run {{toolchain_name}} cargo build`
+
+- Open the local rust documentation in your browser:
```
Comment (CleanMachine1) [inline pages/common/rustup.md:31]:
```suggestion
- Open the local rust documentation the system default browser:
```

File: pages/common/rustup.md [hunk 2]
```diff
@@ -27,3 +27,7 @@
...
+
+`rustup doc`
```
Comment (CleanMachine1) [inline pages/common/rustup.md:33]: Does `rustup documentation` work?

Comment (SebastianJL) [reply pages/common/rustup.md:33]: yes. might depend on the version.

Comment (SebastianJL) [reply pages/common/rustup.md:33]: or do you mean written as 'rustup documentation'? No that doesn't work.

Comment (mfrw) [reply pages/common/rustup.md:33]: Nope, the subcommand is `rustup doc`

Comment (CleanMachine1) [reply pages/common/rustup.md:33]: doc is still intuitive however documentation works better. 

See what overs think before adding it.

File: pages/common/rustup.md (hunk 1 above)
Comment (SebastianJL) [reply pages/common/rustup.md:31]: There is a typo in your suggested change. You forgot the word 'in'.

File: pages/common/rustup.md (hunk 2 above)
Comment (CleanMachine1) [reply pages/common/rustup.md:33]: ok then.

Comment (SebastianJL) [reply pages/common/rustup.md:33]: > doc is still intuitive however documentation works better.
> 
> See what overs think before adding it.

I don't know what you mean. The subcommand `rustup documentation` does not exist.

**Review by CleanMachine1 — APPROVED** [review]
Thanks @SebastianJL

**Review by marchersimon — APPROVED** [review]
Looks good, thanks

**Comment by SebastianJL** [comment]
Thanks for reviewing and approving.

File: pages/common/rustup.md [hunk 3]
```diff
@@ -27,3 +27,7 @@
 - Run cargo build with a certain toolchain:
 
 `rustup run {{toolchain_name}} cargo build`
+
+- Open the local rust documentation in the system default browser:
```
Comment (navarroaxel) [inline pages/common/rustup.md:31]: @marchersimon I think this is better, wdyt?
```suggestion
- Open the local rust documentation in the default web browser:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @SebastianJL, welcome and thanks for your submission. I just leave 1 comment.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**